includes/data-stores/class-wc-admin-reports-variations-data-store.php

```diff
@@ -127,6 +158,9 @@ protected function normalize_order_by( $order_by ) {
 		if ( 'date' === $order_by ) {
 			return $table_name . '.date_created';
 		}
+		if ( 'sku' === $order_by ) {
+			return 'postmeta.meta_value';
```

Comment:
`WC_Admin_Reports_Products_Data_Store` uses just `meta_value` here. I don't think it matters, but any reason not to use the same?

Reply:
Sorry, @psealock I am not understood what you want to explain to me. Can you please elaborate it little more.

Reply:
Hi @psealock , I have tried the same but it was not working if I was using only meta_value. SO, I hav updated as postmeta.metavalue.
As, Comparing to the code both the files have minor structure changes. And it is working fine with postmeta.meta_value also.

Let me know your thoughts on the same.

Thanks & Regards,
Ronak Ganatra

Reply:
Hi @psealock , Updated the changes.